vendor/github.com/redhat-cne/sdk-go/pkg/subscriber/subscriber.go

```diff
@@ -49,7 +49,7 @@ type Subscriber struct {
 	SubStore *store.PubSubStore `json:"subStore" omit:"empty"`
 	// EndPointURI - A URI describing the subscriber link .
 	// +required
-	EndPointURI *types.URI `json:"endPointURI" omit:"empty"`
```

Comment:
is it possible the internal API direct call failure in older version ?

Reply:
no. tested with old version. The subscription was successful.